protobuf_serialization/reader.nim

```diff
@@ -137,6 +183,9 @@ proc readValueInternal[T: object](stream: InputStream, value: var T, silent: boo
             stream.readFieldPackedInto(fieldVar, header, ProtoType)
           else:
             stream.readFieldInto(fieldVar, header, ProtoType)
+        elif ProtoType is ref:
```

Comment:
no `ref` types for now, it's a separate feature that is problematic and not well represented in protobuf in general (circular or multiple ownership in particular)

Reply:
I hid it behind the same conformance flag as the one use for map/Table.
It's mandatory to have something like this for the conformance test, otherwise `import_proto3 test_message_proto3.proto` fails.